src/app/api/schedules/[scheduleId]/route.ts

```diff
@@ -65,7 +66,18 @@ export async function PUT(
       );
     }
 
-    if (updatedGameSession.startTime >= updatedGameSession.endTime) {
+    const parsedStartTime = parse(
+      updatedGameSession.startTime,
+      "dd/MM/yyyy",
+      new Date(),
+    );
+    const parsedEndTime = parse(
+      updatedGameSession.startTime,
+      "dd/MM/yyyy",
+      new Date(),
+    );
+    if (parsedStartTime >= parsedEndTime) {
+      console.log("parsedStartTime >= parsedEndTime");
```

Comment:
I'm not sure if you're editing this right thing here. startTime and endTime are time strings i.e. "09:00:00", not date strings.

This did bring to my attention that certain routes are missing correct validation. I will make a refactor ticket for that but you can delete this change.